### ISSUE TITLE
Consider adding GO15VENDOREXPERIMENT support

### ISSUE THREAD

**Issue by kardianos** [mjibson/party#14]
In 1.5 the go command will read the environment variable GO15VENDOREXPERIMENT. If enabled, it will use the local "/vendor/" folder for dependencies. This replaces the need to rewrite dependencies. While you may not want to completely remove import path rewrites, you may wish to make the vendor folder an option.

Will you consider supporting this approach?

**Comment by mjibson** [comment]
Yes, I would like to support that. I'll probably wait until it's out to work on it, though. Others are free to submit PRs before then.
